src/Options.js

```diff
@@ -247,13 +246,51 @@ export class Options {
     return mismatches;
   }
 
+  transformView() {
```

Comment:
As long as changes are reflected... I'll just wait and see what you had in mind.